server/start.go

```diff
@@ -125,6 +127,7 @@ which accepts a path for the resulting pprof file.
 
 func startStandAlone(ctx *Context, appCreator AppCreator) error {
 	addr := ctx.Viper.GetString(flagAddress)
+	transport := viper.GetString(flagTransport)
```

Comment:
Please no global viper usage -- we spent a large amount of time removing this.

```suggestion
	transport := ctx.Viper.GetString(flagTransport)
```